godel_surface_detection/src/scan/robot_scan.cpp

```diff
@@ -68,51 +63,20 @@ bool RobotScan::init()
 	move_group_ptr_->setEndEffectorLink(params_.tcp_frame);
 	move_group_ptr_->setPoseReferenceFrame(params_.world_frame);
 	move_group_ptr_->setPlanningTime(PLANNING_TIME);
+	move_group_ptr_->setPlannerId("RRTConnectkConfigDefault");
 	tf_listener_ptr_ = TransformListenerPtr(new tf::TransformListener());
 	scan_traj_poses_.clear();
 	callback_list_.clear();
 	return true;
 }
 
-bool RobotScan::load_parameters(std::string ns)
+bool RobotScan::load_parameters(const std::string& filename, const std::string& ns)
 {
-	return load_parameters(params_,ns);;
+	return true;
```

Comment:
Per the PR description, I removed the load/save logic and will replace it shortly with something that people will be happier with.